docs/src/main/asciidoc/extension-authors-guide.adoc

```diff
@@ -494,15 +494,15 @@ public class FileConfig {
     String format;
 
     /**
-     * The file log level.
+     * The level of logs to be written into the file.
      */
     @ConfigItem(defaultValue = "ALL")
     Level level;
 
     /**
-     * The file logging log level.
+     * The name of file where logs will be stored.
```

Comment:
Ah, in fact you spotted it, but you updated the javadoc in the comments of this documentation but not in the original source file :).

Updating the source file is definitely more important.

Reply:
Done.